apps/settings/js/findmydevice.js

```diff
@@ -142,7 +149,6 @@ define('findmydevice', ['modules/settings_utils', 'shared/settings_listener'
     _onCheckboxChanged: function fmd_on_checkbox_changed(event) {
       event.preventDefault();
 
-      var _ = navigator.mozL10n.get;
```

Comment:
if you are removing variable, you must remove/update it's usage, otherwise you will get exception later in code